.eslintrc

```diff
@@ -58,6 +58,22 @@ rules:
   react/no-danger: off # gatsby uses this a lot
   no-use-before-define: ["error", { "functions": false }]
   no-return-await: off
+  # <<< These were enabled when upgrading eslint-config-airbnb 16→19. Disabling for now.
```

Comment:
Historical context -- we started with the eslint-config-airbnb a long time ago, and I remember that the seemed pretty sensible at the time. We haven't regularly updated these (it's a chore) and from time-to-time we would disable rules in this file when we found them to be frustrating. The goal of linting (in my eyes) is largely to help avoid errors / JS gotchas. While stylistic consistency is nice, and linting helps here, certain rules (e.g. `operator-linebreak`) tend to be controversial and not actually improve the code quality all that much so I've tended to turn them off.

Reply:
That makes sense. I think it's still good to keep `eslint-config-airbnb` around (or if not, copy over all the rules that are actively being used). And some of these new rules might be useful in avoiding errors / JS gotchas, but I see that as a separate task to go through rules one-by-one and evaluate usefulness.